public/app/components/settings.js

```diff
@@ -19,6 +19,8 @@ function (_) {
         'singlestat': { path: 'panels/singlestat', name: 'Single stat' },
         'text':       { path: 'panels/text',       name: 'Text' },
         'dashlist':   { path: 'panels/dashlist',   name: 'Dashboard list' },
+        'table-time':   { path: 'panels/table/timeseries',   name: 'Timeseries Table' },
```

Comment:
Sorry, ignore that comment, read it wrong (though it was the require config file)